tests/DCPS/DomainRange/DataReaderListenerImpl.cpp

```diff
@@ -75,7 +73,7 @@ DataReaderListenerImpl::on_data_available(DDS::DataReader_ptr reader)
 
   if (error == DDS::RETCODE_OK) {
     if (info.valid_data) {
-      ACE_DEBUG((LM_DEBUG, "(%P|%t) Domain: %d. %C received message %d from %C.\n", domain, id_.c_str(), message.count, std::string(message.from).c_str()));
+      ACE_DEBUG((LM_DEBUG, "(%P|%t) Domain: %d. %C received message %d from %C.\n", domain_, id_.c_str(), message.count, std::string(message.from).c_str()));
```

Comment:
Not sure if this test is built / used by safety profile, but it looks like the main CPP file is using OPENDDS_STRING to be safe, so do it here too?
```suggestion
      ACE_DEBUG((LM_DEBUG, "(%P|%t) Domain: %d. %C received message %d from %C.\n", domain_, id_.c_str(), message.count, OPENDDS_STRING(message.from).c_str()));
```